Library/Homebrew/utils/diskusage.rb

```diff
@@ -0,0 +1,57 @@
+require "find"
+
+module Utils
+  class DiskUsage
```

Comment:
Yeh, I think we could have a shared method that both `Pathname.abv` and these methods could call. I also think we probably could keep it as a short method inside `utils.rb` rather than requiring a separate class.